### PR TITLE
11-merchant-item-create

### LINKED ISSUE #13: 11. Merchant Item Create
11. Merchant Item Create

As a merchant
When I visit my items index page
I see a link to create a new item.
When I click on the link,
I am taken to a form that allows me to add item information.
When I fill out the form I click ‘Submit’
Then I am taken back to the items index page
And I see the item I just created displayed in the list of items.
And I see my item was created with a default status of disabled.

### PR DESCRIPTION
**Pull request by pcbennett108**
Add tests for new item creation
Add #new and #create to item controller
Add new item creation form

### CODE REVIEW

File: app/controllers/merchants/items_controller.rb [hunk 1]
```diff
@@ -29,6 +29,23 @@ def update
       end
   end
 
+  def new
+    @merchant = Merchant.find(params[:merchant_id])
+    @item = @merchant.items.new
+  end
+
+  def create
+    @merchant = Merchant.find(params[:merchant_id])
```
Comment (brad2412) [inline app/controllers/merchants/items_controller.rb:38]: Looks good!

File: spec/features/merchants/items/edit_spec.rb [hunk 2]
```diff
@@ -32,5 +32,17 @@
       expect(page).to have_content("Name: cheeese")
       expect(page).to_not have_content("Name: cheese")
     end
+
+    it "displays a flash message 'Please fill in all fields' when data entry is invalid" do
+      visit edit_merchant_item_path(@merchant1, @item1)
+  
+      fill_in("Name", with: "")
```
Comment (brad2412) [inline spec/features/merchants/items/edit_spec.rb:39]: Thanks for adding this

**Review by brad2412 — APPROVED** [review]

**Review by algibson1 — APPROVED** [review]

**Comment by algibson1** [comment]
Looking great!

**Review by topher-nullset — COMMENTED** [review]
too good!

**Pull request merged**